btree_tests/ycsb.cpp

```diff
@@ -193,28 +191,30 @@ void ycsb_load_run_randint(int index_type, int wl, int kt, int ap, int num_threa
     std::string init_file;
     std::string txn_file;
 
+    std::string ycsb_dir = "/home/eddy/repo/ycsb";
```

Comment:
can you take this in as an argument instead of hard coding a path to your home directory?